lib/data/app_constants/app_constants.dart

```diff
@@ -6,6 +6,9 @@ class AppConstants {
   static const String postTask = '/create';
   static const String getTask = '/gettask';
 
+  get getTaskUrl => baseUrl + getTasks;
+  get postTaskUrl => baseUrl + postTask;

```

Comment:
変更すべき箇所
- `get` と `post` で分ける
- `getTaskUrl` を `getTasksUrl` に変更
- `getTaskUrl` をgetterに追加

```suggestion
  static const String getTask = '/gettask';
  static const String postTask = '/create';

  get getTasksUrl => baseUrl + getTasks;
  get getTaskUrl => baseUrl + getTask;
  get postTaskUrl => baseUrl + postTask;
```